src/config.js

```diff
@@ -2,6 +2,8 @@
 module.exports = {
   apiKey: process.env.ANVIL_API_KEY,
   apiBaseURL: process.env.API_BASE_URL || 'https://app.useanvil.com',
+  baseURL: process.env.BASE_URL || 'https://esign-demo.useanvil.com',
```

Comment:
This should default to localhost:8080 or whatever so it just works for people who set it up from the readme

Reply:
yeah makes sense